kahuna/package.json

```diff
@@ -32,6 +32,7 @@
       "text": "github:systemjs/plugin-text@^0.0.2",
       "theseus": "npm:theseus@0.5.2",
       "theseus-angular": "npm:theseus-angular@^0.3.1",
+      "titip": "github:quantumui/titip@master",
```

Comment:
Depending on `master` isn't ideal because it makes the builds non-reproduceable, if master is updated. Could you open an issue on that project to request them to tag versions? Alternatively, is it possible to use Git hashes?